src/util/strb.h

```diff
@@ -77,6 +77,15 @@ static inline int strb_error(strb *sb) {
   return sb->l == (size_t)-1;
 }
 
+/*
+ * Initialize at runtime an strb.
+ */
+
+static inline void strb_init(strb* sb){
+  const strb s = STRB_STATIC_INIT;
+  *sb = s;
```

Comment:
@abergeron As a matter of fact, a Release-mode build with GCC recognized that `strb`, along with everything else I set to zero explicitly in `reduxCheckArgs()`, could be memsetted to 0:

```
Dump of assembler code for function GpuArray_reduction:
   0x00007ffff74f5300 <+0>:     push   %r15
   0x00007ffff74f5302 <+2>:     xor    %eax,%eax           # Zero RAX
   0x00007ffff74f5304 <+4>:     push   %r14
   0x00007ffff74f5306 <+6>:     push   %r13
   0x00007ffff74f5308 <+8>:     push   %r12
   0x00007ffff74f530a <+10>:    mov    %rcx,%r12
   0x00007ffff74f530d <+13>:    mov    $0x2f,%ecx          # How many words to clear
   0x00007ffff74f5312 <+18>:    push   %rbp
   0x00007ffff74f5313 <+19>:    mov    %edi,%ebp
   0x00007ffff74f5315 <+21>:    push   %rbx
   0x00007ffff74f5316 <+22>:    sub    $0x1b8,%rsp
   0x00007ffff74f531d <+29>:    test   %r12,%r12
   0x00007ffff74f5320 <+32>:    lea    0x30(%rsp),%rdi     # Base pointer for memset
   0x00007ffff74f5325 <+37>:    lea    0x30(%rsp),%rbx
   0x00007ffff74f532a <+42>:    rep stos %rax,%es:(%rdi)   # Memset to 0.
   0x00007ffff74f532d <+45>:    mov    %ebp,0x30(%rsp)
   0x00007ffff74f5331 <+49>:    mov    %rsi,0x38(%rsp)
   0x00007ffff74f5336 <+54>:    mov    %rdx,0x40(%rsp)
   0x00007ffff74f533b <+59>:    mov    %r12,0x48(%rsp)
   0x00007ffff74f5340 <+64>:    mov    %r8d,0x50(%rsp)
   0x00007ffff74f5345 <+69>:    mov    %r9,0x58(%rsp)
   0x00007ffff74f534a <+74>:    movq   $0x1,0x138(%rsp)
   0x00007ffff74f5356 <+86>:    movq   $0x1,0x150(%rsp)
   0x00007ffff74f5362 <+98>:    movq   $0x1,0x168(%rsp)
   0x00007ffff74f536e <+110>:   movq   $0x1,0x140(%rsp)
   0x00007ffff74f537a <+122>:   movq   $0x1,0x158(%rsp)
   0x00007ffff74f5386 <+134>:   movq   $0x1,0x170(%rsp)
   0x00007ffff74f5392 <+146>:   movq   $0x1,0x148(%rsp)
   0x00007ffff74f539e <+158>:   movq   $0x1,0x160(%rsp)
   0x00007ffff74f53aa <+170>:   movq   $0x1,0x178(%rsp)
```